flightaware_piaware_config/src/flightaware_piaware_config/piaware_config.py

```diff
@@ -160,13 +160,14 @@ def parse(val: str) -> str:
         return val
 
 class MetadataSettings():
-    def __init__(self, processor, default: any = None, setting_type: str = None, protect: str = None, sdonly: bool = None, network: str = None, deprecated = False) -> None:
+    def __init__(self, processor, default: any = None, setting_type: str = None, protect: str = None, sdonly: bool = None, network: str = None, network_manager_value = False, deprecated = False) -> None:
```

Comment:
This seems backwards, there should not be anything NM-specific in the config parsing code; anything NM-specific should live on the output side.